### PR TITLE
Fixes maracas being emaggable

### PR DESCRIPTION
**Pull request by MadmanMartian**
:cl:
 * bugfix: Fixes maracas being emaggable

### CODE REVIEW

**Comment by DeityLink** [comment]
c'mon though, leave the Cuban Pete reference alone.

**Comment by Irradiation** [comment]
No fun allowed

**Comment by MadmanMartian** [comment]
The entire bloody thing is a cuban pete reference.

**Comment by SonixApache** [comment]
fun police is a good tag

**Comment by SonixApache** [comment]
deletethis

**Comment by MadmanMartian** [comment]
Until people stop using the maracas as an excuse to be able to emag Ian/books/carp/the clown

image

**Comment by Riekland** [comment]
unatomic

**Comment by SGuivenne** [comment]
In space nobody can hear you cha cha

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
emagging th singulo when??

**Comment by SGuivenne** [comment]
Dont joke, TK + camera viewer means you actually can muck with the singulo without dying (in theory)

**Comment by bathosbathtime** [comment]
Bugfixing features has gone too far.
On a serious note, I find the PR notes and CL to be too misleading even for a joke.  You have to know that some of the community can't understand code on even the most basic level so they have to take PRs at face value.
But also this is a good change.

**Comment by strawberrytan** [comment]
water potassium

**Comment by MadmanMartian** [comment]
Glad somebody saw the pun

**Comment by Kammerjunk** [comment]
>Until people stop using the maracas as an excuse to be able to emag Ian/books/carp/the clown

Well I dunno. The medigun PR was closed, wasn't it :^)

**Comment by NigglyWiggly** [comment]
"Fixes"

**Comment by Hoshi-chan** [comment]
I understand why you'd do this, but it's really unnecessary.

**Comment by DeityLink** [comment]
Wait what,

>"Fixes maracas being emaggable"
>They're still emaggable

@MadmanMartian what did you mean by this? You even said yourself that the whole bloody item was a Cuban Pete reference, why not remove the bomb part if you were gonna remove the references?

**Comment by DeityLink** [comment]
@Kammerjunk Funny you mention that https://cdn.discordapp.com/attachments/239836068491296788/500864260092461056/dreamseeker_2018-10-14_04-54-16-79.webm

**Comment by MadmanMartian** [comment]
How in the fuck is everyone missing the point?

**Comment by Riekland** [comment]
maybe because only you're autistic enough to see it B^)

**Comment by Gringonius** [comment]
Just merge it, no one will actually care at the change of flavor text.

**Comment by DeityLink** [comment]
How about you explain what the point is then?

**Comment by SonixApache** [comment]
the fix is that it's impossible to turn a piece of wood with beans or beads or whatever in it into an explosive by swiping it with a weird technocard

so instead of being wood and beans, it's a technological device with a battery that could believably be sabotaged

tl;dr autism

**Comment by bathosbathtime** [comment]
Gee r0b0, you are right, why would anyone take your word at face value?  Truly this is a mystery that no one could have seen coming.  We'll have to really look into why this community might not feel it necessary to check the code of one of the collaborators and trust that their code does what they say it does.

**Comment by brndd** [comment]
so what you're saying is that soap could be emaggable if I changed the flavour text of the item?

**Comment by jknpj** [comment]
@brndd we have, as matter of fact, holosoap
